library-best-practices/glossary.md

```diff
@@ -11,7 +14,14 @@ Java Dependency Glossary
   `LinkageError`, or other related exceptions.
   - Sub-type: **Linkage conflict**
   - Sub-type: **Missing class error**
-
+  - Sub-type: **Static linkage error**: A linkage error caused by a direct code
+    reference (non-reflective).
+  - Sub-type: **Dynamic linkage error**: A linkage error caused by a reflective
+    reference.
+
+<a name="linkage-conflict"></a>
+<a name="static-linkage-conflict"></a>
+<a name="dynamic-linkage-conflict"></a>
```

Comment:
should these be closer to their definiti0ons instead of next to each other?

Reply:
Since they are embedded under the made definition, I think it makes sense to anchor at this point.